bool constant_speed
bool accelerate
bool decelerate
bool change_lane
